src/extension/kernel.ts

```diff
@@ -1102,4 +1103,86 @@ export class Kernel implements Disposable {
   public getPlainCache(cacheId: string): Promise<Uint8Array> | undefined {
     return this.serializer?.getPlainCache(cacheId)
   }
+
+  async runProgram(program?: RunProgramOptions | string) {
+    let programOptions: RunProgramOptions
+
+    if (!this.runner) {
+      throw new Error('No runner available')
```

Comment:
Don't think this should be "fatal" and throw. We can recover by logging a warning here and return early. If the `runme.experiments.grpcRunner` is disabled this will make Runme hard stop for no reason.

Reply:
Oh, I see. It’s a leftover from the previous approach. In this case, I think we can return false instead

Reply:
Whatever makes it noop. If the runner is disabled, we should never display a notification.